src/components/exercise/panel.cjsx

```diff
@@ -0,0 +1,100 @@
+React = require 'react'
+_ = require 'underscore'
+
+ArbitraryHtmlAndMath = require '../html'
+Question = require '../question'
+FreeResponse = require './free-response'
+
+{propTypes, props} = require './props'
+panelProps = _.extend {}, propTypes.ExFreeResponse, propTypes.ExMulitpleChoice, propTypes.ExReview
+
+ExPanel = React.createClass
+  displayName: 'ExPanel'
+  propTypes: panelProps
+  getDefaultProps: ->
+    disabled: false
+    free_response: ''
+    answer_id: ''
+  getInitialState: ->
+    {free_response, answer_id} = @props
+
+    freeResponse: free_response
+    answerId: answer_id
+
+  componentDidMount: ->
+    {panel} = @props
```

Comment:
Could this prop be named something like `mode` to clarify the panel could be showing `free-response`, `multiple-choice`, or `review`?